include/circt/Dialect/SV/TypeDecl.td

```diff
@@ -43,6 +43,11 @@ def InterfaceOp : SVOp<"interface",
 
   let extraClassDeclaration = [{
     Block *getBodyBlock() { return &body().front(); }
+
+    ModportType getModportType(::mlir::SymbolRefAttr);
+    ::mlir::FlatSymbolRefAttr getSymRef() {
```

Comment:
Would it make more sense to have a method that returns the `InterfaceType` for the symref, since that is what the clients use?  The `FlatSymbolRefAttr` seems like an internal implementation detail.